openwisp_monitoring/db/backends/influxdb/queries.py

```diff
@@ -2,37 +2,38 @@
     'uptime': {
         'influxdb': (
             "SELECT MEAN({field_name})*100 AS uptime FROM {key} WHERE "
-            "time >= '{time}' AND content_type = '{content_type}' AND "
```

Comment:
This is on home page,
pl. notice the date range and what it is showing: 

(I think I have pointed this out before, are you having difficulties in pin pointing the reason for the problem?)

![dev-01](https://user-images.githubusercontent.com/24985760/185695174-38ef46a7-814d-45fa-b861-093a648f027d.png)

Reply:
Hey @atb00ker, now the general charts work fine in my last pushed commit, and the demo system is not updated with that, I will make sure the build pass asap.  And then we can test if the things work or not.